roadmaps/roadmap_2024.md

```diff
@@ -0,0 +1,118 @@
+# Roadmap - 2024
+
+## Table of contents
+
+- [Roadmap - 2024](#roadmap---2024)
+  - [Table of contents](#table-of-contents)
+  - [Themes for 2024](#themes-for-2024)
+  - [Engineering: Core](#engineering-core)
+  - [Engineering: integrations and extensions](#engineering-integrations-and-extensions)
+  - [User experience](#user-experience)
+  - [Performance and stability](#performance-and-stability)
+  - [Reliability and security](#reliability-and-security)
+  - [Community support and governance](#community-support-and-governance)
+  - [Design](#design)
+  - [General explorations](#general-explorations)
+
+## Themes for 2024
+
+For 2024, we'll continue to focus on the [similar themes as last year](./roadmap_2022.md#themes-for-2022), with stronger focus on engineering tasks.
+We have added several new tasks that are planned for 2024 (marked as 🏗 or 📬) and carried over a few unfinished items from last year.
+
+## Engineering: Core
+
+- 📬 - Improve backup, restore, and upgrade process
+- 🏗 - Enhance account-registration and sign-up mechanism
+- 📬 - Scope and implement a desktop version of Nebari
```

Comment:
Scoped and decided as not-in-scope for Nebari.

Reply:
so
```suggestion
```
?

I just to bring attention back to this PR whenever I was exploring and saw 2024 roadmap missing